aito/sdk/aito_client.py

```diff
@@ -278,6 +279,9 @@ def upload_entries_by_batches(
             batch_size: int = 1000,
             optimize_on_finished: bool = True
     ):
+        """
+        This function will be deprecated in an upcoming release, please use upload_entries instead.
+        """
```

Comment:
```suggestion
        """
        .. deprecated:: 0.2.1
        Use :func:`upload_entries` instead
        """
```
Use sphinx deprecated doc instead